neo/rawio/axonrawio.py

```diff
@@ -451,7 +451,8 @@ def parse_axon_soup(filename):
             f.seek(sections['StringsSection']['uBlockIndex'] * BLOCKSIZE)
             big_string = f.read(sections['StringsSection']['uBytes'])
             goodstart = -1
-            for key in [b'AXENGN', b'clampex', b'Clampex', b'CLAMPEX', b'axoscope']:
+            for key in [b'AXENGN', b'clampex', b'Clampex', b'CLAMPEX',
+                                                                    b'axoscope', b'Clampfit']:
```

Comment:
for key in [b'AXENGN', b'clampex', b'Clampex', b'CLAMPEX',
￼                                                                    b'axoscope', b'Clampfit']: